lebedigital/demonstrator_scripts/beam_design.py

```diff
@@ -52,6 +54,7 @@ def max_bending_moment_and_shear_force(span: float, point_load: float, distribut
             max_shear_force_dist_load + max_shear_force_point_load)
 
 
+@ureg.wraps(None, ('mm', 'mm', 'N*mm', 'N', 'N/mm^2','N/mm^2','mm','mm'))
```

Comment:
Instead of wrapping to mm, could we just continuously use the units also in calling that function?

Reply:
Just realized that stial_diam = int is wrong (not from your merge request)

Reply:
Maybe, but that would require that I understand every step of the function.
I know it works with mm, so I chose to keep it that way. 
The input can still be given in cm,m,km or miles ;)